src/applications/personalization/dashboard/components/notifications/DebtNotification.jsx

```diff
@@ -1,47 +1,34 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import moment from 'moment';
+import { VaAlert } from 'web-components/react-bindings';

```

Comment:
I did just notice that there is a PR open to remove this alias (not sure exactly why its being removed), but might be good to just import directly from node modules via `@department-of-veterans-affairs/component-library/dist/react-bindings` so you don't have to go back and fix this later when that PR gets merged.

PR for reference: https://github.com/department-of-veterans-affairs/vets-website/pull/21127